projects/1.20/assets/tetra/tetra/lang/zh_cn.json

```
@@ -0,0 +1,2465 @@
+  "tetra.stats.ability_revenge": "复仇",
+  "tetra.stats.lunge_revenge.tooltip": "用猛冲攻击复仇目标会重置能力冷却时间",
+  "tetra.stats.lunge_revenge.tooltip_extended": "最近攻击过你的实体被判定为复仇目标，持有具有复仇能力的物品并注视复仇目标时会出现一个HUD指示器，使用复仇能力会重置目标的复仇状态。所有复仇目标在30秒内未被攻击后复仇状态将重置",
+  "tetra.stats.execute_revenge": "苦痛揭示",
+  "tetra.stats.execute_revenge.tooltip": "如果持有者带有负面效果，处决伤害§e+%s%%§r",
+  "tetra.stats.slam_revenge.tooltip": "用猛击击中复仇目标使其眩晕§e%s§r秒",
+  "tetra.stats.slam_revenge.tooltip_extended": "最近攻击过你的实体被判定为复仇目标，持有具有复仇能力的物品并注视复仇目标时会出现一个HUD指示器，使用复仇能力会重置目标的复仇状态。所有复仇目标在30秒内未被攻击后复仇状态将重置",
+  "tetra.stats.puncture_revenge": "逆转",
+  "tetra.stats.puncture_revenge.tooltip": "如果目标比持有者有更多的护甲值，则对目标和持有者同时施加减甲效果和流血效果",
+  "tetra.stats.pry_revenge.tooltip": "撬击对复仇目标造成伤害§e+%s%%（%s）§r并双倍降低护甲值",
+  "tetra.stats.pry_revenge.tooltip_extended": "最近攻击过你的实体被判定为复仇目标，持有具有复仇能力的物品并注视复仇目标时会出现一个HUD指示器，使用复仇能力会重置目标的复仇状态。所有复仇目标在30秒内未被攻击后复仇状态将重置",
+  "tetra.stats.overpower_revenge.tooltip": "压制对复仇目标造成伤害§e+%s%%（%s）§r并且不会使持有者精疲力竭",
+  "tetra.stats.overpower_revenge.tooltip_extended": "最近攻击过你的实体被判定为复仇目标，持有具有复仇能力的物品并注视复仇目标时会出现一个HUD指示器，使用复仇能力会重置目标的复仇状态。所有复仇目标在30秒内未被攻击后复仇状态将重置",
+  "tetra.stats.reap_revenge.tooltip": "收割对复仇目标造成伤害§e+%s%%（%s）§r\n \n§r每次击杀收割伤害§e+1§r，持续§e20§r秒，每击杀一个复仇目标，持续时间§e+%s§r秒",
+  "tetra.stats.reap_revenge.tooltip_extended": "最近攻击过你的实体被判定为复仇目标，持有具有复仇能力的物品并注视复仇目标时会出现一个HUD指示器，使用复仇能力会重置目标的复仇状态。所有复仇目标在30秒内未被攻击后复仇状态将重置",
+  "tetra.stats.ability_echo": "回声",
+  "tetra.stats.lunge_echo.tooltip": "在半空中时，猛冲可以额外使用§e%s§r次，以延长距离或改变方向",
+  "tetra.stats.lunge_echo.tooltip_extended": "在半空中按Shift/潜行键来向后闪避， 按空格/跳跃键（不是右键）来增加向上移动的距离",
+  "tetra.stats.execute_echo.tooltip": "3秒后再次攻击目标，造成基于其已损失生命值和负面效果的伤害",
+  "tetra.stats.slam_echo.tooltip": "3秒后对同一目标二次猛击",
+  "tetra.stats.puncture_echo.tooltip": "3秒后再次刺穿目标",
+  "tetra.stats.pry_echo.tooltip": "3秒后再次撬击目标",
+  "tetra.stats.overpower_echo.tooltip": "持有者的精疲力竭延迟§e%s§r秒生效，再次压制则进一步延迟",
+  "tetra.stats.reap_echo.tooltip": "3秒后对同一区域二次收割\n \n§r首次击杀伤害§e+1§r，持续§e30§r秒，后续击杀伤害§e+%s§r",
+  "tetra.stats.execute": "处决",
+  "tetra.stats.execute.tooltip": "按住右键可以以§e100%%（%s）§f的基础伤害发动一次强力攻击，且伤害会因以下因素进一步提升：\n§7目标每损失1%%的生命值，伤害§e+%s%%§7\n目标每带有1层负面效果，伤害§e+%s%%§7\n \n§7蓄力时间：§e%s§7秒\n§7冷却时间：§e%s§7秒",
+  "tetra.stats.execute.tooltip_extended": "蓄力时间和冷却时间由物品的挥动速度决定，亡灵杀手等效果的额外伤害计入攻击的基础伤害中",
+  "tetra.stats.severing": "切割",
+  "tetra.stats.severing.tooltip": "攻击有§e%s%%§r概率使目标最大生命值§e-10%%§r，攻击伤害§e-5%%§r，持续§e60§r秒，此效果最多叠加§e%s§r层",
+  "tetra.stats.lunge": "猛冲",
+  "tetra.stats.lunge.tooltip": "按住右键蓄力，松开施展猛冲。猛冲击中实体时，会造成§e%s%%（%s）§f的伤害并使能力冷却时间§e-30%%§f\n \n§7蓄力时间：§e%s§7秒\n§7冷却时间：§e%s§7秒",
+  "tetra.stats.lunge.tooltip_extended": "蓄力时间和冷却时间由物品的挥动速度决定，亡灵杀手等效果的额外伤害计入攻击的基础伤害中",
+  "tetra.stats.slam": "猛击",
+  "tetra.stats.slam.tooltip": "按住右键蓄力，松开释放强力猛击\n \n§7猛击地面会从撞击点发出冲击波，被冲击波击中的实体受到§e%s%%（%s）§7的伤害\n \n§7猛击的命中目标会受到§e%s%%（%s）§7的伤害并被强力击退，同时使能力冷却时间§e+50%%§7\n \n§7蓄力时间：§e%s§7秒\n§7冷却时间：§e%s§7秒",

```

| **审查级别**    |
|:-:|
| ❓疑问 / 需要作者确认 |

**原文: Hold right-click to charge up a powerful slam
 
§7Slam the ground to send out a shockwave from the point of impact, entities hit by the shockwave take §e%s%% (%s)§7 attack damage.
 
§7Directly slam an entity to deal §e%s%% (%s)§7 attack damage and cause heavy knockback, this triggers a §e50%%§7 longer cooldown.
 
§7Charge time: §e%s§7 seconds
§7Cooldown: §e%s§7 seconds**

```suggestion
  "tetra.stats.slam.tooltip": "按住右键蓄力，松开释放强力猛击\n \n§7猛击地面会从撞击点发出冲击波，被冲击波击中的实体受到§e%s%%（%s）§7的伤害\n \n§7猛击的命中目标会受到§e%s%%（%s）§7的伤害并被强力击退，同时使能力冷却时间§e+50%%§7\n \n§7蓄力时间：§e%s§7秒\n§7冷却时间：§e%s§7秒",
```